### PR TITLE
Fix Diplomatic Governor display issue on City States Panels

### PR DESCRIPTION
**Pull request by the-m4a**
** What this does **
Corrects the issue where the Diplomat Governor was always shown next to the City State instance (when viewing the list of CS) or next to the list of influencers to a CS.  Documented in #130 

Also fixes issue where the number of Envoys tool tip would not show when CIVITAS City States Expanded was loaded.

**Tested, verified functional**
- CQUI Only: Vanilla, exp1, exp2
- CQUI + CIVITAS: Vanilla, exp2

### CODE REVIEW

File: CQUI.modinfo [hunk 1]
```diff
@@ -338,13 +340,31 @@
...
+                <LoadOrder>230</LoadOrder>
                 <LuaContext>CityStates</LuaContext>
                 <LuaReplace>Assets/UI/PartialScreens/citystates_CQUI_basegame.lua</LuaReplace>
             </Properties>
         </ReplaceUIScript>
 
+        <ReplaceUIScript id="CQUI_CityStates_EXP1"  criteria="Expansion1">
+            <Properties>
+                <!-- Later than usual load to ensure function extensions are loaded -->
+                <LoadOrder>240</LoadOrder>
```
Comment (Infixo) [inline CQUI.modinfo:353]: EXP2 loads @ 200. There is no need to introduce another LO point.

**Comment by Infixo** [comment]
So... the fix is basically to change the name from BASE_xxx to BASE_CQUI_xxx. Ok. But this file is supposed to handle base game and both expansions.
Why do we need separate loaders for exp1 and exp2 suddenly?

**Comment by the-m4a** [comment]
The original change a couple of weeks ago should've had the exp1 and exp2 as well, because Firaxis has their own extensions for the respective expansions, and the CQUI ones need to load after.  They also have the higher load order values to ensure they load after CIVITAS (since we don't have XML) and those files are independent enough that loading them late doesn't appear to affect anything else.

For example: Firaxis' CityStates_Expansion1.lua extends AddCityStateRow, to handle showing or hiding the ambassador.  Since the previous set of changes did not call the firaxis file, the ambassador icon always showed because the CQUI code also didn't handle it.

Also thanks again for the review!

**Comment by the-m4a** [comment]
Also - using BASE_AddCityStateRow as the variable name in CQUI **and** loading after the Firaxis CityStates_Expansion1.lua created a never-ending loop.  The CQUI BASE_AddCityStateRow points to the FiraxisExp1 AddCityState row, which has a first line of calling BASE_AddCityStateRow...

**Comment by the-m4a** [comment]
Lastly - I checked out reverting the load order values to 0/100/250 for base/exp1/exp2 -- the tooltip only works in the exp2 case.  If the load order is all after 201 (Civitas) then the tooltip works in all cases with civitas

**Comment by Infixo** [comment]
Can we please not mix Civitas into cqui? We already discussed it .
If the issue is with CQUI only, then it should be be fixed within CQUI rules and files only.
That means using standard LO points.
I will take a closer at it tomorrow.

**Pull request merged**